Borysov/homework 4. Session, Filters/Quiz/src/main/java/borysov/service/UserService.java

```diff
@@ -0,0 +1,37 @@
+package borysov.service;
+
+import borysov.entity.*;
+import borysov.exception.DAOException;
+
+import javax.jws.soap.SOAPBinding;
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserService {
+
+    static List<User> users = new ArrayList<User>();
+
+
+    static {
+
+        User user1 = new User("den", "11", "mail");
+        User user2 = new User("alex", "alex1", "email");
+        users.add(user1);
+        users.add(user2);
+
+    }
+
+    public User getUserByLoginAndPass(String login, String password) throws DAOException {
+        for (User user : users) {
+            System.out.println(user);
```

Comment:
YOu're using loggers, why do you still need sout?)

Reply:
It was an accident